sample_files/.travis.yml

```diff
@@ -27,4 +30,4 @@ script:
   - travis_run_tests
 
 after_success:
-  coveralls
```

Comment:
Can't we call it the script `coveralls`, that calls the real coveralls command, instead of renaming all Travis?